### ISSUE TITLE
Take down the crossbrand swag shop

### ISSUE THREAD

**Issue by kristoferlund** [commons-stack/iteration0#1112]

**Comment by tamarandom** [comment]
The cross brand shop is not being used. I propose we take it down entirely since it's on mainnet and gas fees are prohibitively high/volatile. 

@iviangita @kristoferlund @GriffGreen Is there any reason we should not take down https://shop.commonsstack.org/#/??

**Comment by kristoferlund** [comment]
No, since mainnet fees prohibits its use we should take it down.

**Comment by GriffGreen** [comment]
yeah kill it.

**Comment by tamarandom** [comment]
OK. Killing it looks like this: (Am I missing anything?)

- Remove dns for shop.commonsstack.org
- Disable the dshop protocol instance running on mainnet

@kristoferlund I can do the first. You good to take on the second?
